Sources/AirbnbSwiftFormatTool/airbnb.swiftformat

```diff
@@ -38,6 +38,7 @@
 --onelineforeach convert #preferForLoop
 --shortoptionals always #typeSugar
 --semicolons never #semicolons
+--doccomments preserve #docComments
```

Comment:
New option added in https://github.com/nicklockwood/SwiftFormat/pull/1638.

Will pull in a new build with support for this option in a few days.